### PR TITLE
Ryanyae 203 mble v project default

### PR DESCRIPTION
**Pull request by ryanyae**
## Description

Finished the mobile view of ProjectDefault
## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

File: src/pages/ProjectV2/Pages/Default/ProjectV2Default.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+const ProjectV2Default = (props: PropOBJ) => {
+  const { project_id } = props;
+  const history = useHistory();
+
+
+  const currentProject:PROJECT | undefined = PROJECT_TEXT.find((element) => element.project_title.toLowerCase() == project_id.toLowerCase());
+  
+  // error handling for projects that do not exist
+  if (currentProject == undefined) {
+    history.push("/404");
```
Comment (ElisaHu1) [inline src/pages/ProjectV2/Pages/Default/ProjectV2Default.tsx:21]: nit: we add a error message here console.error(`Project with ID "${project_id}" not found.`);

**Review by ElisaHu1 — APPROVED** [review]
looks great, I assume this is the first part of the project default change?

**Comment by ryanyae** [comment]
@ElisaHu1 This latest commit is my final version of the PR.

**Comment by ryanyae** [comment]
> looks good to me, do we need an extra ticket for the gallery display?

This one is still work in progress, and I will get this for you by Wednesday.

**Pull request merged**